Source/Core/AudioCommon/WASAPIStream.cpp

```diff
@@ -199,7 +199,7 @@ bool WASAPIStream::SetRunning(bool running)
     // Show a friendly name in the log
     ComPtr<IPropertyStore> device_properties;
 
-    result = device->OpenPropertyStore(STGM_READ, device_properties.GetAddressOf());
+    result = device->OpenPropertyStore(STGM_READ, &device_properties);
```

Comment:
A lot of these were changed from `&` to `.GetAddressOf()` in 8445644e0561469f9216b388ffcc6a8839215095
I'm not sure exactly why.

@cscd98 Is using `&` here fine for mingw?